heron/tools/cli/src/python/response.py

```diff
@@ -33,14 +34,17 @@ class Status(object):
   Ok = 0
   InvocationError = 1
   HeronError = 100
+  DryRun = 200
 
 def status_type(status_code):
   if status_code == 0:
     return Status.Ok
   elif status_code < 100:
     return Status.InvocationError
-  else:
+  elif status_code >= 100 and status_code < 200:
```

Comment:
in your suggestion, HeronError is using number larger than 200 instead of 100, which means we need to modify all the codes that return 100.